consensus/src/simplex/signing_scheme/ed25519.rs

```diff
@@ -56,6 +56,47 @@ impl Scheme {
             signer: None,
         }
     }
+
+    /// Stage a certificate for batch verification.
+    fn batch_verify_certificate<'a, D: Digest>(
+        &self,
+        batch: &mut Batch,
+        namespace: &[u8],
+        context: VoteContext<'a, D>,
+        certificate: &'a Certificate,
+    ) -> bool {
+        // If the certificate does not meet the quorum, return false.
+        if certificate.signers.len() < self.participants.quorum() as usize {
+            return false;
+        }
+        if certificate.signers.len() != certificate.signatures.len() {
```

Comment:
Previous `.zip()` only implementation would've permitted unequal signers/signatures to pass validation (if it made it past `codec::read`